bittide-instances/data/picocom/start.sh

```diff
@@ -8,4 +8,6 @@
 #
 #    /dev/serial/by-id/usb-FTDI_Dual_RS232-if01-port0
 #
-picocom  --baud 921600 --imap lfcrlf --omap lfcrlf $@
+picocom  --baud 921600 --imap lfcrlf --omap lfcrlf $@ \
+  > >(tee -a $PICOCOM_STDOUTLOG) \
+  2> >(tee -a $PICOCOM_STDERRLOG >&2)
```

Comment:
```suggestion
set -u

picocom  --baud 921600 --imap lfcrlf --omap lfcrlf $@ \
  > >(tee -a "${PICOCOM_STDOUTLOG}") \
  2> >(tee -a "${PICOCOM_STDERRLOG}" >&2)
```

Reply:
`set -u`: error if variables are undefined
`${}`: good practice to avoid things like `$FOO_bar`, where you _think_ you're accessing `$FOO`, but you're accessing `$FOO_bar`.
`""`: account for spaces